pkgs/top-level/all-packages.nix

```diff
@@ -18665,6 +18665,15 @@ in
 
   nginx = nginxStable;
 
+  nginxQuic = callPackage ../servers/http/nginx/quic.nix {
+    withPerl = false;
+    # We don't use `with` statement here on purpose!
+    # See https://github.com/NixOS/nixpkgs/pull/10474/files#r42369334
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders ];
```

Comment:
How should I change the quic package in the module? I would need to copy this expression and adapt it right? Maybe we should mention this.

Reply:
Quic currently is part of the HG repository and gets pulled with nginx-quic, there's no seperate module file

Reply:
There's also quiche as alternative quic server, has different options and works via a patch + a seperate rust library linked into nginx.

Reply:
If I wanted to add another module this wouldn't be easily possible right now.

Reply:
https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/web-servers/nginx/default.nix#L404-L416 there's an option for that, also other packages do it like that, too